src/HotCorners/HotCorner.vala

```diff
@@ -38,25 +38,57 @@ public class Gala.HotCorner : Object {
      * In order to avoid accidental triggers, don't trigger the hot corner until
      * this threshold is reached.
      */
+    private const int TRIGGER_PRESSURE_THRESHOLD_MIN = 0;
```

Comment:
Why changing the trigger threshold? This threshold is used the first time the hot corner is activated. With my hardware, it makes difficult to trigger it